tests/performance_tests/performance_test_basic.py

```diff
@@ -275,10 +277,15 @@ def runTpsTest(self) -> PtbTpsTestResult:
 
         self.data.startBlock = self.waitForEmptyBlocks(self.validationNode, self.emptyBlockGoal)
         tpsTrxGensConfig = ltg.TpsTrxGensConfig(targetTps=self.ptbConfig.targetTps, tpsLimitPerGenerator=self.ptbConfig.tpsLimitPerGenerator)
+
+        abiFile = Path("unittests")/"contracts"/"eosio.token"/"eosio.token.abi" if self.ptbConfig.exerciseTrxSpecification else None
+        actionName = "transfer" if self.ptbConfig.exerciseTrxSpecification else None
+        actionData = f'{{"from":"{self.account1Name}","to":"{self.account2Name}","quantity":"0.0001 CUR","memo":"transaction specified"}}'  if self.ptbConfig.exerciseTrxSpecification else None
```

Comment:
I would have expected all of this to be provided on the command line or in some kind of config file.

Reply:
Addressed in updates, most recently: [777d9d3](https://github.com/AntelopeIO/leap/pull/626/commits/777d9d30bf5918a4fb6f106d45eef3f69d3b59f4)